crates/common/tedge_config/src/sudo.rs

```diff
@@ -57,7 +57,7 @@ impl SudoCommandBuilder {
                 c
             }
             Err(_) => {
-                warn!("`sudo.enable` set to `true`, but sudo not found in $PATH");
+                warn!("`sudo.enable` set to `true`, but sudo not found in $PATH, using tedge-write directly");
```

Comment:
`tedge_config::sudo::command()` is not only used by `tedge-write`.

```suggestion
                warn!("`sudo.enable` set to `true`, but sudo not found in $PATH, invoking {program} directly");
```